lib/cequel/metal/keyspace.rb

```diff
@@ -393,7 +408,24 @@ def extract_ssl_config(configuration)
         ssl_config.each { |key, value| ssl_config.delete(key) unless value }
         ssl_config
       end
-
+      
+      def extract_cassandra_error_policy(configuration)
+        value = configuration.fetch(:cassandra_error_policy, ::Cequel::Metal::Policy::CassandraError::ClearAndRetryPolicy)
+        # Accept a class name as a string, create an instance of it 
+        if value.is_a?(String)
+          value.constantize.new
```

Comment:
Let's pass `configuration` under the theory that real soon now we will improve it to be more than just a simple hash.